### DISCUSSION TITLE
Why there is no template with just default pages (index,counter and weather)?!

### DISCUSSION THREAD

**Discussion by AbdussalamAl-shaebi** [bitfoundation/bitplatform#3708] [Ideas]
It very useful and very easy for a lot of project when you have template contain all main blazor projects with blazor default template.
-Blazor WASM Project
-Blazor Server Project
-Blazor MAUI Project
-Shared Project

with default template (index,counter and weather).

**Comment by msynk** [comment]
@AbdussalamAl-shaebi 
Thanks for contacting us. we already started working on adding new simple templates.
you can check out the first one here #3696 
The other templates will be added/enhanced in the near future.
